.ssh/config

```diff
@@ -0,0 +1,6 @@
+Host dev
+HostName 127.0.0.1
+User wrongsecrets
+Port 4444
+# Cha-llen-ge 5-5 (to avoid detection by scanning tools)
+# Se-cret en-cry-pted : U2FsdGVkX18Z71msuvueMq5Tyioi7zYt6FJM/z6qJIvXM8q587ZT4ogr49ccs9Mv
```

Comment:
What if we make this just an ssh pubkey instead? (Or add a pubkey as another challenge)